ringpop.go

```diff
@@ -96,6 +96,25 @@ func mergeDefault(opts *Options) *Options {
 	return opts
 }
 
+type RingpopInterface interface {
```

Comment:
Let's think of a better name for RingpopInterface (I can't think of a better one right now). If there's something more idiomatic out there, let's use it.